src/converse-chatview.js

```
@@ -1018,6 +1026,25 @@ converse.plugins.add('converse-chatview', {
                 return this;
             },
 
+            async inject (ev) {
+                if (!_converse.debug) {
+                    return this;
```

I think this should raise an error, or at least log an error.